ruby/ql/lib/codeql/ruby/libraries/Core.qll

```diff
@@ -0,0 +1,72 @@
+/**
+ * Provides modeling for the Ruby core libraries.
+ */
+
+private import codeql.ruby.Concepts
+private import codeql.ruby.DataFlow
+private import codeql.ruby.dataflow.FlowSummary
+import core.BasicObject::BasicObject
+import core.Object::Object
+import core.Kernel::Kernel
+import core.Module
+import core.Array
+import core.Regexp
```

Comment:
The reason I did this was to reduce churn in other files that previously imported `StandardLibrary.qll` and then used the classes directly. I can add a followup commit that explicitly imports the required modules in those files, and then change these imports back to `import core.Object; import core.Kernel`. Maybe that would be better as a followup, to avoid piling too many things in this PR? What do you think?

Reply:
Having this as a followup sounds good to me.